bumble/profiles/asha_service.py

```diff
@@ -76,8 +76,10 @@ def on_audio_control_point_write(_connection, value):
                     f'volume={value[3]}, '
                     f'otherstate={value[4]}'
                 )
+                self.emit('start', str(value[0]).encode())
```

Comment:
why not emit the 4 arguments as a tuple? (or even a `namedtuple`)?

Reply:
ack